code/lib/cli/src/autoblock/block-react-script-version.ts

```diff
@@ -0,0 +1,26 @@
+import { createBlocker } from './types';
+import { dedent } from 'ts-dedent';
+import { gte } from 'semver';
+
+export const blocker = createBlocker({
+  id: 'storiesMdxUsage',
+  async check({ packageManager }) {
+    const version = await packageManager.getVersion('react-scripts');
```

Comment:
`packageManager.getVersion('react-scripts')` returns the latest version on npmjs, not the installed version.

Reply:
```suggestion
    const version = await packageManager.getPackageVersion('react-scripts');
```